TODO.md

```diff
@@ -0,0 +1,46 @@
+# Things remaining to do for final submission
```

Comment:
This does not belong in the python/cpython repository.  Attach this as a file to the issue.

Reply:
TODO.md has been removed.